NGitLab.Tests/BranchClientTests.cs

```diff
@@ -26,7 +26,7 @@ public async Task Test_CommitInfoIsCorrectlyDeserialized()
             Assert.LessOrEqual(7, commit.ShortId.Length);
 
             var fiveMinutesAgo = DateTime.UtcNow - TimeSpan.FromMinutes(5);
-            Assert.Less(fiveMinutesAgo, commit.CreatedAt);
+            Assert.Less(fiveMinutesAgo, commit.CreatedAt.ToUniversalTime());
```

Comment:
The current version doesn't return UTC date (`DateTime.Kind`)? Maybe we should fix that in another MR if needed

Reply:
I believe that when deserializing `"created_at"`
- SimpleJson converts it to UTC, while
- System.Text.Json keeps it in Local time

Reply:
Added a custom converter & put back test as it was before.